lib/ai/actions/message-actions.ts

```
@@ -42,7 +48,16 @@ export async function saveUserMessage({
 }): Promise<void> {
   // If regeneration, delete the last message
   if (modelParams.isRegeneration) {
-    await convex.mutation(api.messages.deleteLastMessage, { chatId });
+    const { success: deleteSuccess, error: deleteError } =
+      await convex.mutation(api.messages.deleteLastMessage, {
+        serviceKey: process.env.CONVEX_SERVICE_ROLE_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)